saleor/plugins/webhook/tasks.py

```diff
@@ -474,11 +484,88 @@ def send_webhook_request_sync(
 
     attempt_update(attempt, response)
     delivery_update(delivery, response.status)
+    report_event_delivery_attempt(attempt)
     clear_successful_delivery(delivery)
 
     return response_data if response.status == EventDeliveryStatus.SUCCESS else None
 
 
+@app.task
+def observability_send_events():
+    events = buffer.get_events()
+    if not events:
+        return
+    domain = Site.objects.get_current().domain
+    event_type = WebhookEventAsyncType.OBSERVABILITY
+    for webhook in _get_webhooks_for_event(event_type):
+        scheme = urlparse(webhook.target_url).scheme.lower()
+        try:
+            if scheme in [WebhookSchemes.AWS_SQS, WebhookSchemes.GOOGLE_CLOUD_PUBSUB]:
+                for event_num, event in enumerate(events):
```

Comment:
This is only a question: Would be good (and more readable) to wrap this `for` (same for http) inside a helper function that will send webhooks? `observability_send_events` looks like a looong function